sematic/plugins/abstract_kuberay_wrapper.py

```diff
@@ -84,6 +91,36 @@ def _get_ray_version() -> str:
         )
 
 
+@dataclass(frozen=True)
+class AutoscalerConfig:
+    """Configuration for the autoscaler.
+
+    Note that the autoscaler will execute in the same pod with the head node,
+    if the autoscaler is enabled.
```

Comment:
Please add the same resource requirement further clarification, as you did in `ray.md` and below in the error message.